

syntax = "proto3";

import "experimental/beacon_sim/world_map_config.proto";
import "common/math/matrix.proto";

package robot.experimental.beacon_sim.proto;

message MappedLandmarks {
  repeated int32 beacon_ids = 1;
  repeated math.proto.Matrix beacon_in_local = 2;
  math.proto.Matrix cov_in_local = 3;
}
